tests/test_usecase.py

```diff
@@ -391,6 +395,68 @@ def test_wrong_citekey(self):
             self.execute_cmds(cmds)
 
 
+class TestNote(DataCommandTestCase):
+
+    def setUp(self):
+        super(TestNote, self).setUp()
+        init = ['pubs init',
+                'pubs add data/pagerank.bib',
+                ]
+        self.execute_cmds(init)
+        self.note_dir = os.path.join(self.default_pubs_dir, 'notes')
+
+    def test_note_edit(self):
+        cmds = [('pubs note Page99', ['xxx']),
```

Comment:
I think we could have a more interesting and kid-friendly content for our notes in the test...